src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/openapi/EndpointsApiController.java

```diff
@@ -156,6 +156,7 @@ public EndpointsApiController(
         }
         List<ServiceClient> serviceClients = serviceClientConverter
```

Comment:
There are some APIs that return unsorted service client collections:

- `EndpointsApiController.addEndpointServiceClients` (response from adding a new service client to an endpoint)
- `ServicesApiController.addServiceServiceClients` (same for a service)

That's probably OK, although could consider adding the sort to them also, for consistency.

Reply:
Thanks, I added sorting to those endpoints too. I think it's good to try to keep the ordering consistent between different endpoints.